README.md

```diff
@@ -1,37 +1,100 @@
+you should now see your `README.md` file. Click on it to open it in the text
+editor
+6. We can directly type in content here, but we can also use our terminal
+skills to add content.  So, in the terminal, type
+`echo "This is my readme file" > README.md`. If you've got the README file open,
+the new text will appear!
+7. Now that we've got some basic content, we can initialize our local
+repository, so in your terminal, type `git init`.  Your terminal should show
+that an 'empty Git repository' has been initialized.
+8. Type `git add .` to stage the new `README.md` file so it will be tracked by git
+9. Now, type `git commit -m 'initialize git'`.  This will create the first
+commit for this local repository, which will allow us to push our work to the
+remote we created earlier.
+10. Now, make sure you've still got your remote git info copied from GitHub,
+and type the following into the terminal:
+`git remote add origin <your-copied-remote-repository-URL>`. This sets the
+remote, so you can push and pull code.
+
+#### For the Standalone Learn IDE 
+
 1. Create a new directory and add a file. You can run this series of commands:
     * Change into your `code` directory: `cd ~/code`
-      - If your development directory is named something other than `~/code`, that's fine — `cd` into whatever yours is called.
+      - If your development directory is named something other than `~/code`, that's
+      fine — `cd` into whatever yours is called.
     * Create a new directory named `my_new_directory`: `mkdir my_new_directory`
     * Change into the newly-created directory: `cd my_new_directory`
     * Create a new file named `README.md`: `touch README.md`
     * Add some text to the new file: `echo "This is my readme file" > README.md`
-2. `git init`
-3. `git add .` + `git commit -m "initialize git"`. Add and commit the new file created in step 1.
-4. `git remote add origin your-remote-repository-URL`. This sets the remote, so you can push and pull code.
+2. Now that we've got some basic content, we can initialize our local
+repository, so in your terminal, type `git init`.  Your terminal should show
+that an 'empty Git repository' has been initialized.
+3. Type `git add .` to stage the new `README.md` file so it will be tracked by git
+4. Now, type `git commit -m 'initialize git'`.  This will create the first
+commit for this local repository, which will allow us to push our work to the
+remote we created earlier.
+5. Now, make sure you've still got your remote git info copied from GitHub,
+and type the following into the terminal:
+`git remote add origin <your-copied-remote-repository-URL>`. This sets the
+remote, so you can push and pull code.
 
-### Note on Origin
-What is `origin`? "Origin" is simply the default alias assigned to your new remote repo, but we could rename it to anything. Let's try changing the name of the repo to `destination`:
+
+## Use `git remote` to Set the Destination of Your Repo
```

Comment:
You already did that in those steps above. Maybe the "in browser path" and the "local system path" re-join here to `git remote add` and then `git push`.

Reply:
And then mabye this obj. is the breakdown of the `git remote` command